src/API/Google/Settings.php

```diff
@@ -244,15 +228,16 @@ protected function get_options_object(): OptionsInterface {
 	}
 
 	/**
-	 * Create the main shipping service object.
+	 * Create a shipping service object.
 	 *
 	 * @param string $country
+	 * @param string $method
 	 * @param string $currency
 	 * @param mixed  $rate
 	 *
 	 * @return Service
 	 */
-	protected function create_main_service( string $country, string $currency, $rate ): Service {
+	protected function create_shipping_service( string $country, string $method, string $currency, $rate ): Service {
```

Comment:
I guess this was already present in the code, but I was just wondering why we have `$rate` as mixed. In the DB it's explicitly marked as a double. Should we be a bit stricter here.

Reply:
Yes, that confused me as well. I'm going to mark it as a `float` just to be in line with other places.

Reply:
Done in https://github.com/woocommerce/google-listings-and-ads/pull/1171/commits/0e9904992fad55b5763b6fa1f65f47ce4783c7b1